### DISCUSSION TITLE
Issues with connection of two different motion sensors (unresponsive after pairing)

### DISCUSSION THREAD

**Discussion by Jastroger** [Koenkk/zigbee2mqtt#24948] [Devices]
Hi Community,

I'm facing recurring issues with two of my motion sensors, and I'm at a loss for how to resolve them. I’m using Zigbee2MQTT with a ConBee 3 USB stick on a Raspberry Pi 4. My network overall works great, except for these two motion sensors.

What’s puzzling is that the problematic motion sensors are located close to other Zigbee devices (lightbulbs) that are functioning perfectly.

Steps I've taken so far:

- Changed the Zigbee channel from 11 to 20 (I avoided 25, as IKEA devices failed to pair on that channel).
- Deleted (and force-deleted) the motion sensor devices from the network.
- Attempted multiple pair/unpair cycles. While the motion sensors initially pair successfully, they become unresponsive shortly afterward.
- Adjusted the transmitter power settings from 5 to 10, and also tested with no specific value (but this didn’t help).
- Added another IKEA router device
- changed channels of my WIFI

Current setup:

One motion sensor is indoors, and the other is outdoors. Both are near router-capable Zigbee devices like LEDVANCE and IKEA bulbs. However, neither motion sensor works reliably.

Here’s a map of the area where these motion sensors are located:
![image](https://github.com/user-attachments/assets/7f289c26-6d46-48f1-aae3-7b127c796716)

Additional steps:

To troubleshoot further, I purchased another coordinator stick to test as a next step:

SONOFF Zigbee Gateway ZBDongle-E 3.0 USB Dongle Plus (EFR32MG21 + CH9102F/CP2102N Zigbee USB-Stick EFR32MG21 Coordinator).
Despite these efforts, I’m confused why all other devices in my network work well except for these two motion sensors.

Any advice?

If anyone has insights or suggestions on what else I could try, I’d greatly appreciate it.

Best regards,
Jan

P.S. The grayed-out bulb in the image is an offline device, as expected, and is no longer connected.

**Comment by chris-1243** [comment]
It would be great to tell which sensor you are trying to pair. Any errors to report when those sensors are getting unresponive?

On your picture, it seems end device are directly connected to your coordinator. When doing a pairing attempt, choose a soecific router

**Comment by Jastroger** [comment]
I began investigating the issue and attempted to resolve it by pairing the sensors with a HUE/LEDVANCE bulb placed in the same room, approximately 1.5 meters away. Despite the close proximity, this approach did not resolve the issue, as the motion sensors still fail to maintain a connection.

The sensors in question are IKEA Valhorn motion sensors, but I also tested this scenario with Philips motion sensors, encountering the same behavior. I initially suspected that the room itself might be causing interference or blocking the signal. To rule this out, I tested the sensors outdoors, but they also failed to function properly in that environment.

The logs and pairing process show no errors, and the sensors pair successfully during setup. However, after pairing, the sensors never reconnect or establish a stable link. This consistent failure across multiple sensor models and environments suggests an underlying issue I have yet to identify. Any advice or insights would be greatly appreciated.

**Comment by Jastroger** [comment]
Today, I gave it another try and managed to pair a different LEDVANCE bulb in the room. It appears that not all bulbs are compatible with the sensors, but the LEDVANCE E27 White successfully paired.

However, the Valhorn sensors are still not updating their status and remain unresponsive, appearing as "free-floating" on the map. Unfortunately, this means the issue is not yet resolved.
